pom.xml

```diff
@@ -467,21 +452,10 @@
             <id>ossrh-release</id>
             <build>
                 <plugins>
-                    <plugin>
-                        <groupId>org.sonatype.plugins</groupId>
-                        <artifactId>nexus-staging-maven-plugin</artifactId>
-                        <version>1.6.7</version>
-                        <extensions>true</extensions>
-                        <configuration>
-                            <serverId>ossrh</serverId>
-                            <nexusUrl>https://oss.sonatype.org/</nexusUrl>
-                            <autoReleaseAfterClose>false</autoReleaseAfterClose>
-                        </configuration>
-                    </plugin>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-source-plugin</artifactId>
-                        <version>${maven.source.plugin.version}</version>
+                        <version>2.2.1</version>
```

Comment:
Just found personally for some reason having issues with the plugin when I didn't specify it, so I just left it specified.

Reply:
What was the issue? Did you try fixing that? 

Is `maven.source.plugin.version` referenced anywhere else in the file? If it is, how is it working in those other places?

It's strange that only some of these dependencies have issues when referring to versions this way. So, I wonder whether there is any syntax issue or incorrect use here

Reply:
Ended up finding out that this was due to a separate bug issue that I found with the release process, and fixed it up in the internal wiki documentation. This actually does work and fixed it to use the old maven.source.plugin.version